### ISSUE TITLE
Check SSE?

### ISSUE THREAD

**Issue by wgregorian** [nccgroup/Scout2#152]
Was --check-s3-encryption deprecated?

**Comment by l01cd3v** [comment]
Yes, for now at least. I will create a standalone tool that does the same thing but outputs data to a file rather than an HTML report whose DOM size may grow insane unless the bucket is used to store a limited number of objects. Hopefully I'll write the code such that I can reuse it in Scout2 and re-enable this feature.

**Comment by IhorKravchuk** [comment]
Gents, for this feature feel free to use/reuse my tool/code that actually has requested functionality  and doing  S3 bucket level/object audit:
 https://github.com/IhorKravchuk/it-security/blob/master/scripts.aws/s3_enc_check.py 
it does:
- scan AWS account for you buckets and their S3 bucket-level  encryption enforcement status 
- you can interactively or using cli choose bucket to scan for object -level sse encryption status
- shows progress of the scan (very useful for the big buckets)
- output results to the screen/file

**Comment by wgregorian** [comment]
Thanks, @IhorKravchuk I'll give it a try!
